packages/thefatrat/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname=thefatrat
+pkgver=620.af28832
+pkgrel=1
+pkgdesc='TheFatRat a massive exploiting tool: easy tool to generate backdoor and easy tool to post exploitation attack.'
+arch=('x86_64')
+groups=('blackarch' 'blackarch-automation' 'blackarch-exploitation')
+url='https://github.com/mrsmith0x00/TheFatRat'
+license=('GPL3')
+depends=('searchsploit' 'backdoor-factory' 'metasploit' 'xterm' 'bind-tools' 'gcc'
+         'apache' 'gnome-terminal' 'upx' 'ruby-nokogiri' 'openssl' 'imagemagick'
+         'lib32-ncurses' 'lib32-zlib' 'lib32-libstdc++5' 'lsb-release' 'python' 'python-names'
+         'jdk8-openjdk' 'jre8-openjdk' 'unzip' 'mingw-w64-gcc' 'sudo')
+makedepends=('git')
+source=('git+https://github.com/mrsmith0x00/TheFatRat.git')
+install=thefatrat.install
+sha512sums=('SKIP')
+options=('!strip')
```

Comment:
Why?

Reply:
Without this option build produce errors.